common/network/services/udp_receiver.hpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#pragma once
+
+#include <boost/asio.hpp>
+
+#include "common/network/components/receiver.h"
+
+class UdpReceiverService {
+ public:
+  UdpReceiverService(receiver_state_t* const state,
```

Comment:
`clang-format`?

Reply:
It is not triggered, what were you referring to ?